msg/en_US/csql.msg

```diff
@@ -188,6 +191,13 @@ $ csql help messages
    ;HISTORYRead <history_num>   - read entry on the history number into command buffer.\n\
    ;TRAce [ON/OFF] [text/json]  - enable/disable sql auto trace.\n\
    ;SIngleline [ON|OFF]         - enable/disable single-line mode.\n\
+   ;Connect [user-name] [database-name]@[host-name]\n\
+                                - connect user, database, host.\n\ 
+   ;.Hist [ON/OFF]              - start/stop collecting statistics information in CSQL.\n\
+   ;.Clear_hist                 - clear the CSQL statistics information in the buffer.\n\
+   ;.Dump_hist                  - display the CSQL statistics information in CSQL.\n\
+   ;.X_hist                     - display the CSQL statistics information in CSQL\n\
```

Comment:
;.x                            - display the CSQL statistics information in CSQL (or ;.Dump_hist).\n\

Reply:
@kisoo-han 
Would you tell me about `;.x` not `.X_hist`?
I think like set ;Get system_parameter (written in manual ';Get'). We should describe the full name and intuitive meaning.
Furthermore, ;.x is not categorized like any others.